doc/INSTALL.md

```diff
@@ -235,6 +236,34 @@ Build with:
       make PIE=1 DEVELOPER=0 \
       CONFIGURATOR_CC="arm-linux-androideabi-clang -static"
 
+To cross-compile for Raspberry Pi
+--------------------
+
+Obtain the [official Raspberry Pi toolchains](https://github.com/raspberrypi/tools). This document assumes compilation will occur towards the Raspberry Pi 3 (arm-linux-gnueabihf as of Mar. 2018). In addition, obtain and install cross-compiled versions of sqlite 3 and gmp.
```

Comment:
add or change to `git clone https://github.com/raspberrypi/tools.git` for easy follow

Reply:
I think GH users can figure out how to clone the toolchains from the link. :) I just like it when users have access to more info than a simple Git repo address.